modules/integration/src/test/scala/scala/cli/integration/MavenTestHelper.scala

```diff
@@ -0,0 +1,13 @@
+package scala.cli.integration
+
+trait MavenTestHelper {
+
+  protected def mavenCommand(args: String*): os.proc = os.proc(maven, args)
+
+  protected lazy val maven: os.Shellable =
+    Seq[os.Shellable](
+      "mvn",
+      "clean",
+      "install"
```

Comment:
```suggestion
     "mvn", "clean", "compile", "exec:java"
```
Also, unless the main class is defined in the POM (which it maybe could be, if it's defined in a Scala CLI build, this would make defining tests here easier), then we also need the `-Dexec.mainClass` option defined when calling Maven (can be passed in the tests directly)